### PR TITLE
Splite RAG and Chatbot

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
Including:
1. Splite RAG and Chatbot
2. add rerank support
3. update with embedding and rank with langchain interation

image

### CODE REVIEW

File: notebooks/llm-rag-langchain/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+The tutorial consists of the following steps:
+
+- Install prerequisites
+- Download and convert the model from a public source using the [OpenVINO integration with Hugging Face Optimum](https://huggingface.co/blog/openvino).
+- Compress model weights to 4-bit or 8-bit data types using [NNCF](https://github.com/openvinotoolkit/nncf)
+- Create a RAG chain pipeline
+- Run Q&A pipeline
+
+## Installation Instructions
+If you have not installed all required dependencies, follow the [Installation Guide](../../README.md).
```
Comment (eaidova) [inline notebooks/llm-rag-langchain/README.md:22]: notebook has all own deps installed inside, there is no need to point to requirements.txt

I prepared the same changes for other readme files in #1908 
```suggestion
This is a self-contained example that relies solely on its own code.</br>
We recommend running the notebook in a virtual environment. You only need a Jupyter server to start.
For details, please refer to [Installation Guide](../../README.md
```

File: notebooks/llm-chatbot/llm-chatbot.ipynb [hunk 2]
```diff
@@ -296,7 +296,7 @@
     "The command bellow demonstrates basic command for model export with `optimum-cli`\n",
```
Comment (eaidova) [inline notebooks/llm-chatbot/llm-chatbot.ipynb:296]: <p>can we also add reference on rag notebook in this notebook? Something like "next steps"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (OpenVINO-dev-contest) [reply notebooks/llm-chatbot/llm-chatbot.ipynb:296]: you mean in the end of llm-chatbot ?

File: notebooks/README.md [hunk 3]
```diff
@@ -247,6 +246,8 @@
 - [Image-to-Video synthesis with AnimateAnyone and OpenVINO](./animate-anyone/animate-anyone.ipynb)
 - [Lightweight image generation with aMUSEd and OpenVINO](./amused-lightweight-text-to-image/amused-lightweight-text-to-image.ipynb)
 - [Part Segmentation of 3D Point Clouds with OpenVINO™](./3D-segmentation-point-clouds/3D-segmentation-point-clouds.ipynb)
+- - [Create a RAG system using OpenVINO and LangChain](./llm-rag-langchain/llm-rag-langchain.ipynb)
```
Comment (eaidova) [inline notebooks/README.md:249]: please revert, it is automatically generated and will be updated in next commit after merge

**Review by eaidova — APPROVED** [review]

**Pull request merged**